CharlieBackend.Api/Middlewares/IsAccountActiveMiddleware.cs

```diff
@@ -59,32 +51,24 @@ public async Task InvokeAsync(HttpContext context, IAccountService accountServic
 
                     if ((bool)!isActive)
                     {
-                        context.Response.StatusCode = 401;
+                        context.Response.StatusCode = StatusCodes.Status403Forbidden;
 
-                        await context.Response.WriteAsync("Account is not active!");
+                        await context.Response.WriteAsync("Account is not active(i.e. you don't have proper role). You need to authorize!");
```

Comment:
When `isActive == true` it's not about having a "proper" role, or "you need to authorize". It's about account that was deactivated.
IMO a message like "Account is not active!" or "Account is deactivated!" would be suffice.